go/validationlib/basic_check.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2018 The Grafeas Authors. All rights reserved.
```

Comment:
This lib should go under the existing package:
go/v1/api/validators/

Reply:
we also want to support v1beta in favor of OSS community, right? 
if so, shall we still put it under go/v1 ?

Reply:
chatted offline